modules/monitoring-enabling-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-alert-routing-for-user-defined-projects_{context}"]
+= Enabling alert routing for user-defined projects
+
+In {product-title}, an administrator can enable alert routing for user-defined projects.
+This process consists of the following steps:
```

Comment:
If the following steps should be completed in a specific order, consider using an ordered list instead of an unordered (bulleted) list.

Reply:
Thank you for the comment! I was actually thinking about it and even applied it at first, but then I realised the order of these steps does not actually matter, plus it looks better as a bulleted list :D for this reason, I kept it as an unordered list.